ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -134,7 +134,12 @@ fi
 arch_instance_type=$(echo -n "${CONTROL_PLANE_INSTANCE_TYPE}" | cut -d . -f 1)
 BOOTSTRAP_NODE_TYPE=${arch_instance_type}.large
 
-workers=3
+workers=${COMPUTE_NODE_REPLICAS:-3}
+if [[ "${COMPUTE_NODE_REPLICAS}" -le 0 ]]; then
+    workers=0
+elif [[ "${COMPUTE_NODE_REPLICAS}" -gt 3 ]]; then
+    workers=3
```

Comment:
why cap this at 3?

Reply:
The reasoning of the cap was to prevent a case in which someone puts an unreasonably large number of compute nodes, which will incur non-negligible costs.

But if you think I should remove the cap, I'll do it.
Please let me know if I should remove it completely, or increase the cap.

Reply:
I've changed the cap to 5, I think that's a reasonable value.